README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+##### HOMEWORK ASSIGNMENT 1
+I'm **Jiayang He**, a MAPSS student concentrating on *psychology*.
+
+My interest in the area focuses on several specific topics under certain sub-fields of psychology:
+
+* Psychophysiology
+  + Will stress be contagious via watching stressful videos? 
+* Social neuronscience
+  + How social stress impact interoception?
+* Clinical psychology
+  + Will people with depresssion or panic disorder have a different pattern of stress response?
```

Comment:
I like how you use lists to present your research interests, as they are clear and organized.